### ISSUE TITLE
openApp() stucks on Linux after 2.0.5 upgrade

### ISSUE THREAD

**Issue by Alexious-sh** [RaiMan/SikuliX1#438]
Hi!
Version 2.0.5 is unable to start any app, which is blocking the shell. E.g.:
```
openApp("vlc")
print("Done!")
```
It stuck on the first line until you manually close the app. Print will be fired only after that. I guess, it completely breaks app running behavior.

Found related issue #366 - looks like the change there causes the issue. That's strange because I can run such a command without blocking the terminal:
```
sh -c "vlc" &
```
PS: I'm running Ubuntu 20.04 if it makes sense.

**Comment by RaiMan** [comment]
Could you please try, wether 

`run('sh -c "vlc" &')`

does the job?

**Comment by Miksser** [comment]
Hi. I tried use simple code
```
run('sh -c "vlc"') # with & i tried too 
topHalfScreen = Region(0,0,1200,800)
topHalfScreen.wait("1.png").click()
```

but the process hangs forever

**Comment by Alexious-sh** [comment]
Yep. The same for me. The process just hangs until I close the app.

**Comment by RaiMan** [comment]
Ok, thanks.

I will to get an Ubuntu 20 to running the next days.

So I can test and evaluate myself.

**Comment by balmma** [comment]
This is most probably all fixed with PR #418 where all this argument splitting is done properly and the sh wrapper is removed entirely. Probably worth porting this back to the 2.0.x branch or pushing 2.1 further?
